demo/c/test_demo.py

```diff
@@ -0,0 +1,143 @@
+#
+# Copyright 2021 Picovoice Inc.
+#
+# You may not use this file except in compliance with the license. A copy of the license is located in the "LICENSE"
+# file accompanying this source.
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+# an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+#
+
+import logging
+import os
+import platform
+import subprocess
+import sys
+import unittest
+
+from pvporcupine import util
+
+class PorcupineDemoTestCase(unittest.TestCase):
+    @staticmethod
+    def __has_detected(s):
+        return "detected" in s
+
+    @staticmethod
+    def __append_language(s, language):
+        if language == 'en':
+            return s
+        return f'{s}_{language}'
+
+    @classmethod
+    def setUpClass(cls):
+        cls.__ENVIRONMENT = cls.__get_environemnt()
+
+    def setUp(self):
+        self.command = ["./build/porcupine_demo_file",
+                    "-l", util.pv_library_path('.'),
+                    "-m", util.pv_model_path('.'),
+                    "-k", "",
+                    "-t", "0.5",
+                    "-w", "",
+                    "-a", sys.argv[1]]
+
+    def test_americano(self):
+        self.command[6] = self.__pv_keyword_path("en", "americano")
```

Comment:
instead of using indices (which would be a pain to keep track of over time), you should just use a utility function that takes in named arguments, inserts defaults where appropriate, and returns the command list.